css/at-rules/document.json

```diff
@@ -94,11 +82,25 @@
               "edge": {
                 "version_added": false
               },
-              "firefox": {
-                "version_added": "6"
-              },
+              "firefox": [
+                {
+                  "version_added": "61",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "layout.css.moz-document.content.enabled",
```

Comment:
Considering this feature has been behind a flag for so long, it's probably not going to be enabled by default.  Since I'm apparently adding the flag here, I'm going to keep it removed for simplicity.